lib/fog/core/model.rb

```diff
@@ -21,16 +21,7 @@ def initialize(new_attributes = {})
     end
 
     def inspect
-      Thread.current[:formatador] ||= Formatador.new
-      data = "#{Thread.current[:formatador].indentation}<#{self.class.name}"
-      Thread.current[:formatador].indent do
-        unless self.class.attributes.empty?
-          data << "\n#{Thread.current[:formatador].indentation}"
-          data << self.class.attributes.map { |attribute| "#{attribute}=#{send(attribute).inspect}" }.join(",\n#{Thread.current[:formatador].indentation}")
-        end
-      end
-      data << "\n#{Thread.current[:formatador].indentation}>"
-      data
+      Fog::Formatador.new(self)
```

Comment:
Or `display`, format doesn't imply stdout to me per se. Open to suggestions.

I'd probably just create a new formatador instance per call. Again, the thread safety issue probably won't come up, but this will limit the window of possible issue.